java/ingest/ingest-batcher-job-creator/src/test/java/sleeper/ingest/batcher/job/creator/IngestBatcherJobCreatorLambdaIT.java

```diff
@@ -107,13 +107,13 @@ private IngestJob readJobMessage(Message message) {
     }
```

Comment:
Please migrate to SQS SDK v2 to receive messages from the queue for the test assertion.